receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md

```diff
@@ -9,6 +9,7 @@ These are the metrics available for this scraper.
 | Name | Description | Unit | Type | Attributes |
 | ---- | ----------- | ---- | ---- | ---------- |
 | **process.cpu.time** | Total CPU seconds broken down by different states. | s | Sum(Double) | <ul> <li>state</li> </ul> |
+| process.cpu.utilization | Percentage of total CPU time used by the process since last scrape. On the first scrape, no data point is emitted for this metric. | 1 | Gauge(Double) | <ul> <li>state</li> </ul> |
```

Comment:
I assume the value is a fraction of 1 not actual percents, right? I believe it's the same confusing description in `system.cpu.utilization` which we would need to correct